This definition causes the following tests to effectively turn no-op,
resulting in strict-multilib violations on amd64.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,8 +18,6 @@
     SET(LINUX TRUE)
 ENDIF ()
 
-SET (LIB_SUFFIX "")
-
 # Pre-set LIB_SUFFIX for 64-bit Linux platforms
 IF (LINUX AND CMAKE_SHARED_LIBRARY_PREFIX STREQUAL "lib64")
   SET (LIB_SUFFIX "64")
